tools/ocean_data_view_manager/tool_odv.xml

```diff
@@ -0,0 +1,188 @@
+                    qc_convention: '${config.qc_convention}' 
+                tool_option:
+                    subsetting: '${config.subsetting}' 
+                    plt: '${config.plt}' 
+    #end if 
+        </configfile>
+    </configfiles>
+    <inputs>
+        <param name="input_raw" type="data" format="netcdf" label="Input raw data"/>
+        <param name="input_ref" type="data" format="netcdf" optional="true" label="Input reference data"/>
+        <param name="mode" type="select" label="Select in which mode you want to run the tool">
+            <option value="R_APP_QV">QV that creates a odv collection with raw and ref files to qualify</option>
+            <option value="DEMO">Valide with ODV demo that performes the demo</option>
+        </param>
+        <conditional name="config">
+            <param name="type" type="select" label="Select if you want to write your own configuration file or not.">   
+                <option value="no">No, I don't want to, I'll use the default one</option>
+                <option value="yes">Yes, I to write my own configuration file</option>
+            </param>
+            <when value="yes">
+                <param name="operator" type="text" value="Anonymous" label="Enter operator name"/>
+                <param name="qc_convention" type="select" label="Enter QC convention for the ODV collection output (default: ARGO)">
+                    <option value="ARGO">ARGO</option>
+                    <option value="ODV">ODV</option>
+                    <option value="GTSPP">GTSPP</option>
+                    <option value="SEADATANET">SEADATANET</option>
+                    <option value="ESEAS">ESEAS</option>
+                    <option value="WOD">WOD</option>
+                    <option value="WODSTATION">WODSTATION</option>
+                    <option value="WOCEBOTTLE">WOCEBOTTLE</option>
+                    <option value="WOCECTD">WOCECTD</option>
+                    <option value="QARTOD">QARTOD</option>
+                    <option value="BODC">BODC</option>
+                    <option value="PANGAEA">PANGAEA</option>
+                    <option value="SMHI">SMHI</option>
+                    <option value="OceanSITES">OceanSITES</option>
+                    <option value="IODE">IODE</option>
+                </param>
+                <param name="subsetting" type="float" min="-1" max="1" value="1" label="Enter subsetting (default: 1)"/>
+                <param name="plt" type="float" min="0" max="1" value="0" label="Enter plt (default: 0 for QV, 1 for demo)"/>
+            </when>
+            <when value="no">
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" label="ODV tool output"/>
+        <collection type="list" name="files" label="ODV tool collection">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)" directory="./" recurse="true" format="txt"/>
```

Comment:
```suggestion
            <discover_datasets pattern="(?P&lt;designation&gt;.+)" directory="./outputs" recurse="true" format="txt"/>
```

Reply:
What are those files? Is it useful to have them as a collection? Can we merge them into one file?